README.md

```diff
@@ -210,6 +210,27 @@ steady_state :step, scopes: false do
 end
 ```
 
+### Prefixed Scopes
+
+On ActiveRecord objects, you may optionally define a prefix for your scope.  This may be useful when dealing with multiple state machines on one object.
+
+```ruby
+steady_state :state, scopes: true do
+  state 'solid', default: true
+  state 'liquid', from: 'solid'
+  # ...
+end
+
+steady_state :temperature, scopes: true, scopes_prefix: 'temperature' do
```

Comment:
```suggestion
steady_state :temperature, scopes: { prefix: true } do
```

I might propose this style (similar to Rails' `delegate` prefix option), where `prefix: true` would default to the name of the state field, and `prefix: 'something_else'` would allow you to override it with something else.

Reply:
Great!  And also, I really like the idea to make it like rails delegate..  When Ray and I were talking about this, he also thought about `prefix: true` but we didn't make the connection to do it all three ways.  I will update the PR and write a description shortly!